packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -6,6 +6,7 @@ import { useId } from 'react-id-generator';
 import useDidUpdate from '../../utils/use-did-update';
 
 type TabsProps = {
+  id?: string;
```

Comment:
While transitioning the old test from Enzymes to RTL and rewiring the entire code to align with RTL best practices, I gave it a go. Discovered a couple of hiccups in Line 220 and 334 related to the ID, but I sorted it out by adding the "id" to TabsProps.

Reply:
hmm I don't see `id` listed as a prop in [the documentation for this component](https://cauldron.dequelabs.com/components/Tabs#tabs), however it looks like the old tests had a couple cases for this. @scurker is this a feature that we'd want to support and hence add to the docs? Or should we not add the prop and remove the tests for "customized id"?

Reply:
I think this component existed before we implemented typescript, which is probably why that initially slipped through. Changing of props would change this from `test` to `fix`, so I think it would be fine to omit those tests for now and not include the id as looking at the tests I'm scratching my head trying to figure out what we were actually testing for.

Reply:
Makes sense to me! @lsprr could you please remove the id prop that you added as well as the tests for "customized id"?